borg/testsuite/convert.py

```diff
@@ -0,0 +1,97 @@
+import os
+import pytest
+import shutil
+import tempfile
+
+try:
+    import attic.repository
+    import attic.key
+    import attic.helpers
+except ImportError:
+    attic = None
+pytestmark = pytest.mark.skipif(attic is None,
+                                reason = 'cannot find an attic install')
+
+from ..converter import AtticRepositoryConverter, NotImplementedException, AtticKeyfileKey
+from ..helpers import get_keys_dir
+from ..key import KeyfileKey
+from ..repository import Repository, MAGIC
+from . import BaseTestCase
+
+class ConversionTestCase(BaseTestCase):
+
+    def open(self, path, repo_type  = Repository, create=False):
+        return repo_type(os.path.join(path, 'repository'), create = create)
+
+    def setUp(self):
+        self.tmppath = tempfile.mkdtemp()
```

Comment:
so this is related to [pytest fixtures](http://pytest.org/latest/fixture.html). i have *tried* to make this work, but somehow i can't get the subs in the class to call the fixture, even if the fixture is global. maybe i'm doing something wrong, but the tests work as they are now and i don't want to bother with this further to remove two lines of code...